services/metadata_service/api/utils.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+from functools import wraps
+
+import pkg_resources
+import collections
+from aiohttp import web
+from multidict import MultiDict
+
+from services.utils import get_traceback_str
+
+version = pkg_resources.require("metadata_service")[0].version
+METADATA_SERVICE_VERSION = version
+METADATA_SERVICE_HEADER = 'METADATA_SERVICE_VERSION'
+
+ServiceResponse = collections.namedtuple("ServiceResponse", "response_code body")
+
+def format_response(func):
+    """handle formatting"""
+
+    @wraps(func)
+    async def wrapper(*args, **kwargs):
+        db_response = await func(*args, **kwargs)
+        return web.Response(status=db_response.response_code,
+                            body=json.dumps(db_response.body),
+                            headers=MultiDict(
+                                {METADATA_SERVICE_HEADER: METADATA_SERVICE_VERSION}))
+
+    return wrapper
+
+
+def web_response(status: int, body):
+    return web.Response(status=status,
+                        body=json.dumps(body),
+                        headers=MultiDict(
+                            {"Content-Type": "application/json",
+                             "Access-Control-Allow-Origin": "*",
+                             METADATA_SERVICE_HEADER: METADATA_SERVICE_VERSION}))
```

Comment:
as this is now only used for the metadata service responses, we can probably remove the access-control headers as unnecessary.